source/developers-guide/plugin-extension-by-plugin/index.md

```diff
@@ -0,0 +1,750 @@
+---
+layout: default
+title: Enhance a plugin with a plugin
+github_link: developers-guide/plugin-extendion-by-plugin/index.md
+indexed: true
+tags:
+  - extend
+  - plugin
+  - custom
+  - products
+---
+
+<div class="toc-list"></div>
+
+## Introduction
+This document will describe how existing plugins can be extended.
+
+Why change a plugin with a plugin: Because changes after a plugin update are gone.
```

Comment:
Why change a plugin with a plugin? To keep the changes compatible if the plugins gets an update